app/src/org/odk/collect/android/widgets/GeoPointWidget.java

```diff
@@ -60,100 +59,86 @@ public GeoPointWidget(Context context, FormEntryPrompt prompt) {
 
         mWaitingForData = false;
         mUseMaps = false;
-        mAppearance = prompt.getAppearanceHint();
+        String appearance = prompt.getAppearanceHint();
+        // use maps or not
+        if (appearance != null && appearance.equalsIgnoreCase("maps")) {
```

Comment:
Jenny reminded me the other day its more succinct to write "maps.equalsIgnoreCase(appearance) which handles nullity for you